tests/test_encrypted_cookie_storage.py

```diff
@@ -5,69 +5,94 @@
 
 import pytest
 from aiohttp import web
+from aiohttp.web_middlewares import _Handler
+from aiohttp.test_utils import TestClient
+
+from typing import no_type_check, Any, Dict, Tuple, Union
 
 from cryptography.fernet import Fernet
 
 from aiohttp_session import (Session, session_middleware, get_session,
                              new_session)
 from aiohttp_session.cookie_storage import EncryptedCookieStorage
 
+from typedefs import _TAiohttpClient
+
 
 MAX_AGE = 1
 
 
-def make_cookie(client, fernet, data):
+def make_cookie(
+    client: TestClient,
+    fernet: Fernet,
+    data: Dict[Any, Any]
+) -> None:
     session_data = {
         'session': data,
         'created': int(time.time())
     }
 
     cookie_data = json.dumps(session_data).encode('utf-8')
-    data = fernet.encrypt(cookie_data).decode('utf-8')
+    encrypted_data = fernet.encrypt(cookie_data).decode('utf-8')  # type: str
```

Comment:
Inline, but should be inferred (unless `cryptography` is not typed).

Reply:
Was indeed inferred, removed the annotation :)